src/test/java/org/jitsi/meet/test/MuteTest.java

```diff
@@ -159,29 +159,25 @@ public void ownerMutesParticipantAndCheck()
 
         WebElement elem = owner.findElement(By.xpath(
             TestUtils.getXPathStringForClassName("//span", "remotevideomenu")
-            + "/i[@class='icon-thumb-menu']"));
+            + "//i[@class='icon-thumb-menu']"));
```

Comment:
(In other words, I agree but not sure I should do it at this point for consistency.)